openedx_proversity_reports/utils.py

```diff
@@ -102,57 +99,19 @@ def populate_children(block, all_blocks):
         """
         children = block.get('children', [])
 
+        block_type = block.get("type")
+
+        counter[block_type] = counter.get(block_type, -1) + 1
```

Comment:
This sentence depends of block_type, if it's None, will fail, please, make sure that this block of code only works if block_type is not None. @andrey-canon